qCC/ccPointPairRegistrationDlg.cpp

```diff
@@ -52,8 +52,8 @@
 
 //default position of each columns in the to-be-aligned and ref. table widgets
 static const int XYZ_COL_INDEX			= 0;
-static const int RMS_COL_INDEX			= 3;
-static const int DEL_BUTTON_COL_INDEX	= 4;
+static const int RMS_COL_INDEX			= 6;
```

Comment:
So I understand you added 2 columns. Isn't it too crowded now? Can you post a snapshot?

Reply:
i made the columns a little bit smaller and the widget little bit larger. 

On a full hd monitor with 100% scaling it is fine. i have a 1280x720 there the widget maybe too big, but not really changed much. It was already taking huge amount of space there. 

new FHD
![image](https://github.com/CloudCompare/CloudCompare/assets/10095691/d48540f8-772a-4b1a-9bab-c890f3412ef7)

old FHD
![image](https://github.com/CloudCompare/CloudCompare/assets/10095691/11b09131-451f-41ce-b508-117e11aa8be8)